# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../aufs2/userspace.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./include/linux/aufs_type.h.orig        2000-01-01 00:00:00.000000000 +0100
+++ ./include/linux/aufs_type.h     2009-09-18 13:32:04.000000000 +0200
@@ -49,6 +49,10 @@
 #endif
 #endif /* __KERNEL__ */

+#ifndef KERNEL
+#define __user
+#endif /* !__KERNEL__ */
+
 /* ---------------------------------------------------------------------- */

 #define AUFS_NAME              "aufs"

